server/db/models/Category.js

```diff
@@ -0,0 +1,40 @@
+const Sequelize = require('sequelize');
+const db = require('../db');
+
+const Category = db.define('category', {
+  name: {
+    type: Sequelize.STRING,
+    allowNull: false
```

Comment:
Do you need a validate: {notEmpty: true}?

Reply:
done